src/test/java/com/example/bak/user/domain/UserRepositoryStub.java

```diff
@@ -22,4 +22,9 @@ protected boolean isSame(Long left, Long right) {
     public Optional<Profile> findProfileByUserId(Long userId) {
         return findById(userId).map(User::getProfile);
     }
+
+    @Override
+    public Optional<User> findByEmail(String email) {
+        return store.stream().filter(u -> Objects.equals(u.getEmail(), email)).findFirst();
```

Comment:
```suggestion
        return store.stream()
                .filter(u -> Objects.equals(u.getEmail(), email))
                .findFirst();
```

메서드 체이닝 과정에서는, 특히 stream이면 이후 체이닝은 줄을 바꿔주는게 가독성이 좋습니다!